src/PolyhedralGeometry/Polyhedron/standard_constructions.jl

```diff
@@ -1187,14 +1159,14 @@ function rand_spherical_polytope(d::Int, n::Int; distribution::Symbol=:uniform,
   else
     throw(ArgumentError("rand_spherical_polytope: invalid distribution specified"))
   end
-  opts = Dict{Symbol,Any}( :template_parameters => [type] )
+  opts = Dict{Symbol,Any}( :template_parameters => [type] ) # creating the Optionset, the :template_parameters are for templating functions in C++
   if seed != nothing
```

Comment:
```suggestion
  if seed !== nothing
```
Please check for `nothing` using `===` and `!==` instead of `==` and `!=`. (Reference: https://docs.julialang.org/en/v1.6/manual/performance-tips/#Checking-for-equality-with-a-singleton)

Reply:
There are more places in `standard_constructions.jl`

Reply:
Why remark this here? There are places all over Oscar. And how would our normal devs know that `nothing` is a singleton and that they should look there? Also why recommend these checks and not use Julia's `isnothing`?